src/lib/containers/download_list/DownloadListTable.tsx

```diff
@@ -188,25 +187,25 @@ export default function DownloadListTable(props: DownloadListTableProps) {
     }
   }
 
-  const filesToDownload = (downloadList && downloadList.filesToDownload) || []
-  const results = (references && references.results) || []
+  const filesToDownload = downloadList?.filesToDownload ?? []
+  const results = references?.results ?? []
   let numBytes = 0
   let numFiles = 0
-  return (
-    <div>
-      <div className="SRC-split download-list-table-top">
-        <span className="create-package-text SRC-centerContentInline">
+  const content = (
+    <div className="DownloadListTable">
+      <div className="SRC-split download-list-table-top SRC-primary-background-color SRC-border-bottom-only">
```

Comment:
I am thinking that here and in a similar situation it might make sense to use BEMs  instead of having a long list of clases. e.g. here we can have <div className = DownloadListTable__top that would have all the properties that we need . The in .scss file we'd have 
`DownloadListTable 
  $__top {
         all the props we need
`

Reply:
I agree with that! I have this issue https://sagebionetworks.jira.com/browse/PORTALS-1199 to track that